README.md

```diff
@@ -2,9 +2,9 @@
 List of top ten psychologists:
 
 1. B. F. Skinner
-1. Zigmund Fraud
+1. Zygmunt Fraud
 1. Albert Bandara
-1. Jean Piaget
+1. Sean da Piaget
```

Comment:
I thought he was french...

Reply:
No, Sean Piaget Ryan Francis Henriques (born 9 January 1973), is a Jamaican hip hop recording artist.